portsyncd/portsyncd.cpp

```diff
@@ -134,6 +135,13 @@ int main(int argc, char **argv)
                     p.set("PortInitDone", attrs);
                     SWSS_LOG_NOTICE("PortInitDone");
 
+                    /*
+                     * Notify "System ready" feature that we are ok.
+                     */
+                    FieldValueTuple app_state("up_status", "true");
```

Comment:
Why is portinitdone alone considered for sysready? In case of warmboot/fastboot scenario should we consider reconciliation done as sys ready?